README.md

```diff
@@ -16,11 +16,16 @@ The polyfill will only be loaded if the URLPattern doesn't already exist on the
 // CJS module loading (Node.js)
 require("urlpattern-polyfill");
 
-// ESM module loading (Node.js)
+// ESM module loading (Node.js and browser)
+if (!globalThis.URLPattern) {
+  await import("urlpattern-polyfill");
+}
+/** 
+ * The above is the recommend way to load the ESM module. 
```

Comment:
* The above is the recommended way to load the ESM module, as it only 
 * loads it on demand, thus when not natively supported by the runtime or 
 * already polyfilled.